packages/zosfiles/src/api/methods/download/Download.ts

```diff
@@ -96,7 +96,7 @@ export class Download {
             IO.createDirsSyncFromFilePath(destination);
 
             if ((!options.binary) && (content.toString().length > 0)) {
```

Comment:
Since we're in this area, I would suggest doing `content.byteLength` over `content.toString().length`.